src/StyleableBannerWithLink/index.tsx

```diff
@@ -117,20 +142,28 @@ const StyleableBannerWithLink: React.FC<Props> = (props: Props) => {
                             <strong css={styles.highlight}>{highlight}</strong>
                         </p>
                     ) : null}
+
                     <div css={styles.primaryButtonWrapper}>
-                        <LinkButton
-                            href={buttonUrl}
-                            css={styles.primaryButton}
-                            onClick={() =>
-                                trackClick({
-                                    internalButtonId: 0,
-                                    ophanComponentId: ophanComponentId as string,
-                                })
-                            }
-                        >
-                            {buttonText}
-                        </LinkButton>
-                        {showPaymentIcons === 'true' && <PaymentIcons />}
+                        <ContributionCtaButton
```

Comment:
I wonder if it's confusing to use a component `ContributionCtaButton` here, since this banner is more generic than this (it could be a link to anything marketing want to promote)?

Reply:
Shall we change the component's name to something like `PrimaryCtaButton`?

Reply:
That sounds great 👍